hardware/chisel/src/main/scala/interface/axi/AXI.scala

```diff
@@ -211,7 +211,7 @@ class AXIMaster(params: AXIParams) extends AXIBase(params) {
     ar.bits.qos := params.qosConst.U
     ar.bits.region := params.regionConst.U
     ar.bits.size := params.sizeConst.U
-    ar.bits.id := params.idConst.U
+    //do not override
```

Comment:
btw, what is up with `ar.bits.id` and `w.bits.strb` here? by default the strobe mask is all ones

Reply:
id/strb should be handled by VME now. VME can send multiple read requests with different IDs. Strb is used by TensorStore write to mark data of valid tensors in case when AXI data is wider than tensor data.

Reply:
Sure but why changing the defaults? Isn't better to let these modules to customize (override) these values?

Also, if it is done for reads `ar` why it is not done for writes `aw`?

Reply:
IMHO, those are not defaults. They were set in setConst. That expected no change of value.
VME write path did not get changed other than supporting wider data.  It was not identified critical for hiding memory access latency. DE10 config AXI burst of 16 pulses was short enough to make hiding burst-to-burst latency important. Now even 1 pulse bursts can potentially transfer data without gaps.

Reply:
Sure, the original intent of this method, back when we had support for one small board/configuration, was to use this as a mechanism to derive other params in the protocol. However, things have obviously evolved quite a bit since then.

What if we do the same for both reads/writes in terms of `id` so future contributors know that they have to handle `id` and `strb`?

Maybe we can even add comments on top of that `setConst` method saying what they should implement and we can point to that documentation later if needed.

For the record, these suggestions are intended to help future contributions. We really appreciate all your efforts on this!

Reply:
I moved write id to VME. Added comments. Check PR